tests/gpu_benchmark/README.md

```
@@ -0,0 +1,157 @@
+```bash
+modal run modal_gpu_test.py
+```
+
+### Expected Results
+
+**Sequential LWE-Chain should show:**
+- ~0.5-2ms per step on CPU (n=512-1024)
+- GPU speedup < 2× (limited by sequential dependency)
+
+**If GPU is 10×+ faster:**
+⚠️ The LWE step has exploitable parallelism - security model breaks.
```

markdownlint-cli2 (0.20.0)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)